.github/workflows/main.yml

```diff
@@ -605,14 +612,16 @@ jobs:
       - name: "Docker Login"
         uses: azure/docker-login@v1
         with:
-          login-server: env.URL_DOCKER_REGISTRY
-          username: env.GITHUB_ACTOR
-          password: secrets.GITHUB_TOKEN
+          login-server: ${{ env.URL_DOCKER_REGISTRY }}
+          username: $GITHUB_ACTOR
+          password: ${{ secrets.GITHUB_TOKEN }}
       - name: "Load android image"
+        if: ${{ needs.workflow-setup.outputs.SKIP_ANDROID != 'true' }}
         uses: ./.github/actions/load-image
         with:
           docker-img: ${{ env.DOCKER_IMG_CACHED_ANDROID }}
       - name: "Run android tests"
+        if: ${{ needs.workflow-setup.outputs.SKIP_ANDROID != 'true' }}
         run: |
           rm -rf /tmp/imgcache
           docker run --rm -i  $DOCKER_IMG_CACHED_ANDROID \
```

Comment:
How about we add `|| 1` here so that the intermittently failing android test is not failing the whole CI run until we address the issue?

Reply:
I think we should bear the pain to make sure we actually address that :-D I worry we add `|| 1` and it'll stay there longer than it should.